ocaml/testsuite/tests/typing-layouts/basics_beta.ml

```diff
@@ -1,8 +1,9 @@
 (* TEST
-   flags = "-extension layouts_beta"
```

Comment:
Why does this PR delete some `*_beta.ml` tests but not others?

Deleted:
  * `ocaml/testsuite/tests/typing-layouts/annots_beta.ml`
  * `ocaml/testsuite/tests/typing-layouts-float64/basics_alpha_beta.ml`
  * `ocaml/testsuite/tests/typing-layouts/datatypes_beta.ml`

Not deleted:
  * `ocaml/testsuite/tests/typing-layouts/basics_beta.ml`
  * `ocaml/testsuite/tests/typing-layouts/modules_beta.ml`

In all of these cases, the `*_beta.ml` tests were added to the non-beta version of the tests.

Reply:
Because ocamltest doesn't allow you to have different output for different configuration of expect-tests (except for around `-principal`). Some of the output changes depending on what extension flag you have enabled. So they have to remain in different test files.

Reply:
OK, that makes sense.